Source/Controllers/SPAppController.m

```diff
@@ -311,15 +311,15 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification {
 - (void)addCheckForUpdatesMenuItem {
     if (NSBundle.mainBundle.isMASVersion == NO && [[NSUserDefaults standardUserDefaults] boolForKey:SPShowUpdateAvailable] == YES) {
         SPLog(@"Adding menu item to check for updates");
-        NSMenuItem *updates = [[NSMenuItem alloc] initWithTitle:NSLocalizedString(@"Check for updates", @"menu item Check for updates") action:@selector(checkForNewVersionFromMenu) keyEquivalent:@""];
+        NSMenuItem *updates = [[NSMenuItem alloc] initWithTitle:NSLocalizedString(@"Check for Updates...", @"Menu item Check for Updates...") action:@selector(checkForNewVersionFromMenu) keyEquivalent:@""];
```

Comment:
Should we use "…" instead of "..."?

Reply:
Ideally, yes. But this follows the convention of the other menu items in that menu. There is a mix of "…" and "..." throughout the project, maybe it can be standardized...